app/views/appointment_summaries/confirm.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="page-header">
+  <h1><%= title 'Confirm appointment summary' %></h1>
+</div>
+
+<div class="row">
+  <div class="col-md-8">
+    <p class="alert alert-warning">
+      Check the customer details below and edit or confirm.
+    </p>
+
+    <table class="table lead">
+      <tbody>
+        <tr>
+          <td class="active" width="35%"><b>Customer name</b></td>
+          <td><%= @appointment_summary.full_name %></td>
+        </tr>
+        <% if @appointment_summary.requested_digital %>
+          <tr>
+            <td class="active" width="35%"><b>Email</b></td>
+            <td><%= @appointment_summary.email %></td>
+          </tr>
+        <% end %>
+        <% unless @appointment_summary.requested_digital %>
+          <tr>
+            <td class="active" width="35%"><b>Address</b></td>
+            <td>
+              <address><%= @appointment_summary.postal_address %></address>
+            </td>
+          </tr>
+        <% end %>
+        <tr>
+          <td class="active" width="35%"><b>Date of appointment</b></td>
+          <td><%= @appointment_summary.date_of_appointment.to_s(:govuk_date) %></td>
+        </tr>
+        <tr>
+          <td class="active" width="35%"><b>Guider Name</b></td>
+          <td><%= @appointment_summary.guider_name %></td>
+        </tr>
+      </tbody>
+    </table>
+    <div class="confirm-submit-buttons">
+      <%=
+        submit_button_classes = %w(btn-primary t-confirm)
+        submit_button_classes << 'js-save-and-open' unless @appointment_summary.requested_digital?
+
+        render 'preview_form', appointment_summary: @appointment_summary,
+               submit_button_text: 'Confirm summary document', submit_button_classes: submit_button_classes
+      %>
+
+      <%= render 'preview_form', url: new_appointment_summary_path, method: :get, appointment_summary: @appointment_summary,
+               submit_button_text: 'Edit summary document', submit_button_classes: %w(btn-danger t-back) %>
+    </div>
+  </div>
+</div>
+
+<script>
+  <%
+    path = if @appointment_summary.persisted?
+             appointment_summary_path(@appointment_summary, format: :json)
+           else
+             appointment_summaries_path(format: :json)
+           end
+  %>
+  $(".js-save-and-open").click(function(e){
+    var popupWindow = window.open('<%= creating_appointment_summaries_path %>');
+
+    $.post('<%= path %>', $(this.form).serialize())
```

Comment:
Not my code 🏃 - have changed quotes - but, i'm sure it could do with a further tidy up and move to the javascript folder 👍